OoyalaSkinSampleApp/OoyalaSkinSampleApp/Lists/MultiAudioListViewController.m

```diff
@@ -0,0 +1,38 @@
+//
+//  MultiAudioListViewController.m
+//  OoyalaSkinSampleApp
+//
+//  Copyright © 2018 Ooyala, Inc. All rights reserved.
+//
+
+#import "MultiAudioListViewController.h"
+
+
+@interface MultiAudioListViewController ()
+
+@end
+
+
+@implementation MultiAudioListViewController
+
+- (void)addTestCases {
+  
+  
+//  [self addCommonWithTitle:@"HLS - 2 tracks: English, German" embedCode:@"M5YjVwZTE6oW_pqUYlrv7gVhecA1rU-G" pcode:@"BzY2syOq6kIK6PTXN7mmrGVSJEFj"];
+//  [self addCommonWithTitle:@"HLS - 2 tracks: Und(2 chan), Und(2 chan)" embedCode:@"w1YjVwZTE6pB5YN6yhkzK_VD3C62CXtm" pcode:@"BzY2syOq6kIK6PTXN7mmrGVSJEFj"];
+//  [self addCommonWithTitle:@"HLS - 2 tracks: Und(2 chan), Und(6 chan)" embedCode:@"93YTVwZTE6w78UPpPX7kh5Q29yuCPBsq" pcode:@"BzY2syOq6kIK6PTXN7mmrGVSJEFj"];
+//  [self addCommonWithTitle:@"HLS - 2 tracks: Und(2 chan), Und(1 chan)" embedCode:@"BvYTVwZTE6Q2BktlPzOkAbRIUv_MLTox" pcode:@"BzY2syOq6kIK6PTXN7mmrGVSJEFj"];
+//  [self addCommonWithTitle:@"HLS - 4 tracks: Eng(2), und(1), Ger(2), und(1)" embedCode:@"AwbjRwZTE61zl6Vg1ubU7hYDmW2SFJlA" pcode:@"BzY2syOq6kIK6PTXN7mmrGVSJEFj"];
+//  [self addCommonWithTitle:@"HLS - 4 tracks: Eng(2, main), Eng(2, commentary), Ger(2, main), Ger(2, commentary)" embedCode:@"xqbTRwZTE6FQy_NWl5-fOOWEDWLPUn5V" pcode:@"BzY2syOq6kIK6PTXN7mmrGVSJEFj"];
+//  [self addCommonWithTitle:@"HLS - 6 tracks: Eng(2, main), Eng(2, commentary), Ger(2, main), Ger(2, commentary), Eng(2), Ger(2)" embedCode:@"pkbTRwZTE6jSHcfUNzSJH122Xt5AVJMP" pcode:@"BzY2syOq6kIK6PTXN7mmrGVSJEFj"];
+  
+  
+  [self addCommonWithTitle:@"MT-Variant-1" embedCode:@"wxNjRwZTE6bxMBkrQbfM_BwokGf0pyOm" pcode:@"k0a2gyOt0QGNJLSuzKfdY4R-hw2b"];
```

Comment:
Let's try to add meaningful names. If the assets are the same in iOS and Android sample apps, use the same name for them. You can sync with @AlinaVoronkovaEpam since she is doing Android and decide on the names you want to use.